.github/workflows/conda.yml

```diff
@@ -28,11 +28,21 @@ jobs:
             - name: Install dependencies
               run: |
                   conda install -n test-env numpy scipy pytest pytest-cov
-                  conda install -n test-env pytorch torchvision cpuonly -c pytorch
+
+                  if [ "${{ matrix.python-version }}" = "3.9" ]; then
+                      channel="-c=conda-forge"
```

Comment:
okay, maybe it's an idiot comment but we do a "-c pytorch" in order to install pytorch via its repo. Woudln't it be good?

Reply:
Added a little comment explaining.